src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/DefaultValueResolver.php

```diff
@@ -27,7 +27,7 @@
      */
     public function supports(Request $request, ArgumentMetadata $argument)
     {
-        return $argument->hasDefaultValue() || $argument->isNullable();
+        return $argument->hasDefaultValue() || ($argument->isNullable() && !$argument->isVariadic());
```

Comment:
but variadics allow null: `function foo(...$foo) {};` => `foo(null)` is allowed, that's the point here.